services/web-server/README.md

```diff
@@ -104,7 +104,7 @@ Variables:
     "deadline": "2018-03-07T06:03:06.683Z",
     "expires": "2019-03-07T06:03:06.683Z",
     "payload": {
-      "image": "ubuntu:13.10",
+      "image": "ubuntu:latest",
```

Comment:
not ubuntu:jammy?

Reply:
Nah, these are just examples in the README. Fine to leave as latest.

Reply:
Better to have our examples set to latest instead of v13 lol